src/firefly/js/templates/lightcurve/LcPeriodogram.jsx

```diff
@@ -256,7 +257,7 @@ class PeriodogramOptionsBox extends Component {
                         </button>
                         <br/>
                         <div style={{marginTop: 10}}>
-                            {'Fields of "Period Min", "Period Max" and "Step Size" are allowed to be blank.'}
+                            {'Leave the fields blank to use default values.'}  <HelpIcon helpId={'periodogram.options'}/>
```

Comment:
This is not a FormPanel component. Plus, I will get more review on those things next week and will do a ticket for all anyway. Prefer to differ the change next week.